plugins/outputs/opensearch/sample.conf

```diff
@@ -0,0 +1,103 @@
+# Configuration for OpenSearch to send metrics to.
+[[outputs.OpenSearch]]
+  ## URLs
+  ## The full HTTP endpoint URL for your OpenSearch instance. Multiple URLs can
+  ## be specified as part of the same cluster, but only one URLs is used to
+  ## write during each interval.
+  urls = ["http://node1.os.example.com:9200"]
+
+  ## Index Name
+  ## The target index for metrics (OpenSearch will create if it not exists).
+  ## You can use the date specifiers below to create indexes per time frame.
+  ## The metric timestamp will be used to decide the destination index name
+  ##   %Y - year (2016)
+  ##   %y - last two digits of year (00..99)
+  ##   %m - month (01..12)
+  ##   %d - day of month (e.g., 01)
+  ##   %H - hour (00..23)
+  ##   %V - week of the year (ISO week) (01..53)
+  ## For example: telegraf-%Y.%m.%d would set it to telegraf-2006-01-02
+  index_name = ""
+
+  ## Default Index Tag Value
+  ## Additionally, you can specify a tag name using the notation {{tag_name}}
+  ## which will be used as part of the index name: "telegraf-{{host}}-%Y.%m.%d"
+  ## If the tag does not exist, the default tag value will be used.
```

Comment:
I would mention the possibility to use tags in the relevant section of "Index Name" and here just what happens if that tag doesn't exist..

Reply:
It will be replaced with empty sting, have explained.
Just so you know- don't see a scope of handling the default_tag_value anymore with this implementation(have tried in test-cases, it only replaced with "" and not the value specified for DefaultTagValue), so removed DefaultTagValue field from struct. Let me know if I'm mistaken here and it can still work.